src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard/health/health.component.ts

```diff
@@ -62,26 +64,25 @@ export class HealthComponent implements OnInit, OnDestroy {
 
   preparePoolUsage(chart, data) {
     const colors = [
```

Comment:
Why not using a colour generator/random colours? Even better if it copes with color blindness (8% population affecting).

- I've seen that the [AngularJS ChartJS](http://jtblin.github.io/angular-chart.js/) module includes a random generator.
- A solutions based on [palette.js](https://stackoverflow.com/questions/39871319/automatic-colors-assignment-in-chart-js-2-x-doesnt-work-anymore-used-to-work-i) and [another](http://google.github.io/palette.js/).
- And [here](https://github.com/chartjs/Chart.js/issues/815) you can find a discussion on this.

Reply:
Thanks, I will take a look.